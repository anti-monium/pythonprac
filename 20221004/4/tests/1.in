"[x for i in range(0, x)]", "x**2", "x + [y]"
11
