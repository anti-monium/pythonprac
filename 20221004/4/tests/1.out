[11, 11, 11, 11, 11, 11, 11, 11, 11, 11, 11, 121]
